package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "date-fp",
-  "version": "3.0.5",
+  "version": "3.0.6",
```

Comment:
Yeah, I can see it both ways. It's certainly unfortunate that someone depending on `date-fp@3.x.x` and relying on the 2.0.0–3.0.5 behaviour will have her code broken by this change.